src/WallEvaluatorColloid.h

```diff
@@ -196,6 +253,6 @@ class WallEvaluatorColloid
 
     } // end namespace detail
     } // namespace azplugins
+    } // end namespace hoomd
 
-#undef DEVICE
```

Comment:
You need to undef both `DEVICE` and `HOSTDEVICE` here or they pollute the header space.